content/posts/survey-launch/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "UC OSPO Network Launches Open Source Survey"
+date: 2025-04-16
+draft: false
+description: "The UC OSPO Network is conducting a survey to understand the open source practices and needs of the University of California community."
+tags: [network, survey]
+displayInList: true
+author: ["Virginia Scarlett"]
+
+resources:
+  - name: featuredImage
+    src: "firmbee-com-unsplash-small.jpg"
+    params:
+      description: "Image courtesy of firmbee.com via Unsplash"
```

Comment:
You might consider using the description to add alt text for visually impaired folks. I really like this graphic to help with composing alt text, and if you're interested in learning more there's a great [blog post on accessibility](https://www.perkins.org/resource/how-write-alt-text-and-image-descriptions-visually-impaired/)!

![image](https://github.com/user-attachments/assets/51510ab8-5203-4f25-84e6-6ef9c1225a9a)